msdb/msdb/static/css/project.css

```diff
@@ -65,3 +65,18 @@
   transform: scaleX(1);
   transform-origin: bottom left;
 }
+
+
+footer {
+  margin-top: 4em;
+}
+
+body{ 
+  min-height: 100vh;
+  display:flex; 
```

Comment:
keep the styling consistent, add a space after colon
display: flex;
similar comment line 77 and 81